build/commands/lib/logging.js

```diff
@@ -124,6 +124,26 @@ function command (dir, cmd, args) {
   status(`${cmdArrowStyle('>')} ${cmdCmdStyle(cmd)} ${args.join(' ')}`, true)
 }
 
+function printLogFailuresToJson(allPatchStatus, bravePath) {
```

Comment:
nit: this isn't any log failures, this is patch failures. IMO the function name should be explicit because this isn't Typescript and the function params aren't explicit.

Reply:
Done, renamed to `printFailedPatchesInJsonFormat`.